keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -664,14 +725,15 @@ reason.
 
 ### New PodConditions
 
-The following condition types are introduced to account for different
-reasons for pod termination (we focus on covering these scenarios were the new
+A new condition type, called `TargetedForDisruption`, is introduced to indicate
+a pod failure caused by external disruption. In order to account for different
+reasons for pod termination we add the following reason fields based on the
```

Comment:
```suggestion
reasons for pod termination we add the following reason types based on the
```
since you're not adding fields, but rather new reason types, right?